Libraries/LibWeb/ResourceLoader.h

```diff
@@ -41,7 +41,7 @@ class ResourceLoader : public Core::Object {
 public:
     static ResourceLoader& the();
 
-    void load(const URL&, Function<void(const ByteBuffer&)>);
+    void load(const URL&, Function<void(const ByteBuffer&, const Optional<String>)>);
```

Comment:
Why don't we pass two callbacks, one for happy case and one for errors?

Reply:
Thanks for the suggestion - done :)